programs/axelar-solana-its/src/lib.rs

```diff
@@ -490,6 +496,115 @@ pub fn deployment_approval_pda(
     }
 }
 
+/// Tries to create the PDA for a [`Tokenmanager`] using the provided bump,
+/// falling back to `find_program_address` if the bump is `None` or invalid.
```

Comment:
```suggestion
/// falling back to `find_program_address` if the bump is `None`.
```

Falling back to `find_program_address` in case of invalid bump would not be desirable in certain situations, like in the lines 572-580 of this file.